experimental/framework/data-objects/src/signalManager/README.md

```diff
@@ -0,0 +1,91 @@
+**_Documentation is a work-in-progress_**
+
+## SignalManager
```

Comment:
If you have concrete known followups on the documentation this is fine to leave in, but otherwise not needed.

```suggestion
# SignalManager
```